usagov/usagov-2021
# USAGOV-1891-wizard-step-indicator: Add an optional step indicator field to the wizard.

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1891

## Description

Add field_step_number configs. Update wizard twig and scss to display the field.

## Type of Changes

- [ X ] New Feature
- [ X ] Frontend (Twig, Sass, JS)
<img width="834" alt="image" src="https://github.com/user-attachments/assets/02259cef-4e17-4dbd-ba0a-2c8e54ffe875">

## Testing Instructions


1. Navigate to any wizard page with options.
2. Inspect the DOM at the main heading (the `<h1>`).
3. You don't see any element in between the `<h1>` and the `<div>`.
4. Edit the page you are on.
5. In the Step Number field, fill in an integer value. Save.
6. The desktop and mobile view of the page have the word "Step" followed by the integer you put in step 5. The design for this element is more or less exactly like the [figma prototype](https://www.figma.com/design/sznp6ZqUSiuRBDYcoiL1PO/FY24-Scam-Wizard-Mobile%2BDesktop-Prototype?node-id=144-19658&m=dev).
7. Switch to a Spanish page and repeat. In step 6 the word "Step" is replaced with "Paso".

### Change Requirements



### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
8. Find the commit of this pull request.
9. Build and deploy the changes.
10. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
